src/callinput.c

```diff
@@ -210,6 +213,26 @@ char callinput(void)
 		break;
 	    }
 
+	case 17:	// CTRL+q
+	    {
+		if (qtcdirection & 1) {
+		    qtc_main_panel(RECV);
+		}
+		if (qtcdirection == 2) {
+		    qtc_main_panel(SEND);
+		}
```

Comment:
No. If qtcdirection is 3, that means the (qtcdirection & 1) will be evaulated as TRUE, but (because qtcdirection = 3), the (qtcdirection == 2) will be false. That's works as well in the code.
When I tested the last code with the waedc-based config, I've found some small disfunction, but I think that's not affects this.